tests/databrowser.js

```diff
@@ -430,6 +430,7 @@ define([
 
             it("should error for unknown properties", function(done) {
                 widget.setValue(escapedDobProp);
+                widget.handleChange();
```

Comment:
I don't think `setValue` should call `handleChange` since it's what's used when initializing a widget, so when we use setValue in a test context like this, we should call it explictly. 

@millerdev & @orangejenny does this sound correct?